src/components/modes/teacher/Settings.js

```diff
@@ -74,37 +62,32 @@ const Settings = () => {
         <Fab
           data-cy={SETTINGS_BUTTON_CY}
           color="primary"
-          className={classes.fab}
           onClick={(event) => {
             event.stopPropagation();
             handleModalOpen();
           }}
+          sx={{ bottom: 1, right: 1, position: 'fixed' }}
         >
           <SettingsIcon />
         </Fab>
-        <Modal
-          open={modalOpen}
-          onClose={handleModalClose}
-          className={classes.modal}
-        >
-          <div data-cy={SETTINGS_CY} className={classes.modalContainer}>
-            <Typography className={classes.header}>{t('Settings')}</Typography>
+        <StyledModal open={modalOpen} onClose={handleModalClose}>
+          <ModalContainer data-cy={SETTINGS_CY}>
+            <Typography sx={{ fontSize: '1.5vw' }}>{t('Settings')}</Typography>
             <ImageUpload />
             <BackgroundToggle />
             <DownloadActions />
-            <Divider className={classes.divider} />
-            <div className={classes.buttonContainer}>
-              <Button
+            <Divider sx={{ mt: 2, mb: 2 }} />
```

Comment:
`mt: 2, mb: 2` can be abreviated to `my: 2` (setting the margin on the y-axis using the short-hand property name)